_includes/manuals/1.24/1.2_release_notes.md

```diff
@@ -34,7 +34,10 @@ systemctl enable --now dynflow-sidekiq@{orchestrator,worker}
 
 #### Cockpit single sign-on
 
-Cockpit single sign-on via "Web Console" button. For more information how to set it up, head over to [Remote Execution](/plugins/foreman_remote_execution/1.7/index.html#3.6CockpitIntegration) manual.
+Cockpit single sign-on is now available via the "Web Console" button on the host view page when Remote Execution is installed and configured for it. For more information how to set it up, head over to [Remote Execution](/plugins/foreman_remote_execution/1.7/index.html#3.6CockpitIntegration) manual.
+
+#### New status API endpoints
+Two new API endpoints have been introduced, `api/ping` and `api/status`. The former provides a way of ensuring Foreman is up and running and requires no authentication. The latter requires authentication and provides additional details on the versions and statuses of various services connected to Foreman. These should be useful for users wanting to monitor the status of their Foreman server. ([#19731](https://projects.theforeman.org/issues/19731))
```

Comment:
@ofedoren does this properly describe the api endpoints you added?

Reply:
@tbrisker, the second endpoint should be `api/statuses` (in plural form).

Otherwise, yes, fully :)

Reply:
Thanks, fixed

Reply:
Note we already had `/status` (not in `/api` which gives quite minimal JSON response about the system. I think that only checks the database though.